From 9536426d4e12b107a706611ead246a9e0ea1d628 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bernhard=20Rosenkr=C3=A4nzer?= <bero@lindev.ch>
Date: Sat, 20 Apr 2024 22:25:18 +0200
Subject: [PATCH] Fix plugin loader (#2052)

The lxqt-plugin binary needs to export symbols that should be visible to
plugins being loaded with QPluginLoader or dlopen.
---
 panel/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/panel/CMakeLists.txt b/panel/CMakeLists.txt
index 556d2212f..d9fe784bf 100644
--- a/panel/CMakeLists.txt
+++ b/panel/CMakeLists.txt
@@ -91,6 +91,8 @@ lxqt_translate_ts(QM_FILES SOURCES
 
 lxqt_app_translation_loader(SOURCES ${PROJECT_NAME})
 
+set(CMAKE_EXECUTABLE_ENABLE_EXPORTS TRUE)
+
 add_executable(${PROJECT}
     ${PUB_HEADERS}
     ${PRIV_HEADERS}
